doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -71,34 +76,22 @@ Traceability to Reference Model.
 | `req.inf.stg.02` | Storage | The Architecture **should** provide Object storage for VM Instances. | OpenStack Swift Service ([RA-1 4.3.1.4 "Swift"](./chapter04.md#4314-swift)) |
```

Comment:
From my understanding, Swift is mandatory in RA1 Chapter 5 and then it may be fixed here and switch to **must**.
Setting as optional must be discussed all together.

Reply:
A RM change #524 just went through to mark Object Storage services as optional.